src/Runner.Worker/ActionManager.cs

```diff
@@ -117,6 +131,22 @@ public async Task<List<JobExtensionRunner>> PrepareActionsAsync(IExecutionContex
                             imagesToBuildInfo[setupInfo.ActionRepository] = setupInfo;
                         }
                     }
+
+                    var repoAction = action.Reference as Pipelines.RepositoryPathReference;
+                    if (repoAction.RepositoryType != Pipelines.PipelineConstants.SelfAlias)
```

Comment:
when RepositoryType == SelfAlias does that mean it's a local action (i.e. `uses: ./[...]`)?

Reply:
yes